cli/flox-activations/src/activations.rs

```diff
@@ -80,7 +82,10 @@ impl Activations {
             anyhow::bail!("activation for store path '{store_path}' already exists");
         }
 
-        let id = Uuid::new_v4();
+        let mut chars = blake3::hash(store_path.as_bytes()).to_hex();
+        // We need something short to put in socket paths
+        chars.truncate(8);
```

Comment:
(non-blocking) I'm not 100% sure whether this is safe from collisions, but the low risk for the number of activations of a single environment doesn't seem worth figuring out.

Reply:
wasn't it something like `sqrt(16^8) ~~ #hashes that collide with a prob of 50%`
for reference in this case that would be about 64k

Reply:
I'm not 100% sure of the justification for why `chars.truncate` is safe, so maybe we should factor that out and comment

But yeah I think those odds are good enough for now